new-site/src/docs/elements/components/checkbox/code.mdx

```diff
@@ -3,21 +3,17 @@ slug: '/elements/components/checkbox/code'
 title: 'Checkbox - Code'
 ---
 
-import { Link } from 'hds-react';
-
-import { Card, SelectionGroup } from 'hds-react';
-import { IconAngleDown, IconAngleUp, IconCheckCircleFill, IconCrossCircle } from 'hds-react';
+import { IconAngleDown, IconAngleUp, IconCheckCircleFill, IconCrossCircle, Link, SelectionGroup } from 'hds-react';
 
 import Playground from '../../../../components/Playground';
 
-## Code
+## Codes
```

Comment:
This seems to be just Code in other component pages. Do we want to just repeat the tab name here?

Reply:
👍  I'll fix it back
- [x] revert heading change